sdk/modelsrepository/azure-iot-modelsrepository/README.md

```diff
@@ -1,186 +1,30 @@
+This is the Microsoft Azure MyService Management Client Library.
+This package has been tested with Python 3.6+.
+For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
 
-## Getting started
+## _Disclaimer_
 
-### Install package
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
 
-Install the Azure IoT Models Repository library for Python with [pip][pip]:
+# Usage
 
-```Shell
-pip install azure-iot-modelsrepository
-```
 
-### Prerequisites
-* A models repository following [Azure IoT conventions][repo_conventions]
-    * The models repository can be hosted on the local filesystem or hosted on a webserver
-    * Azure IoT hosts the global [Azure IoT Models Repository][global_azure_repo] which the client will use if no custom location is provided
+To learn how to use this package, see the [quickstart guide](https://aka.ms/azsdk/python/mgmt)
 
-### Publishing Models
-Follow the [guide](https://docs.microsoft.com/azure/iot-pnp/concepts-model-repository#publish-a-model) to publish models to the global Azure IoT Models Repository.
 
-If using a custom local or remote repository, you can simply add your model files to a directory structure in the repository location, e.g. `dtmi/com/example/thermostat-1.json`
+ 
+For docs and references, see [Python SDK References](https://docs.microsoft.com/python/api/overview/azure/)
+Code samples for this package can be found at [MyService Management](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com.
+Additional code samples for different Azure services are available at [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
 
-### Authentication
-Currently, no authentication mechanisms are supported. The global endpoint is not tied to an Azure subscription and does not support authentication. All models published are meant for anonymous public consumption.
 
-## Key concepts
+# Provide Feedback
 
-The Azure IoT Models Repository enables builders to manage and share digital twin models. The models are [JSON-LD][json_ld] documents defined using the Digital Twins Definition Language ([DTDL][dtdl_spec]).
+If you encounter any bugs or have suggestions, please file an issue in the
+[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
+section of the project. 
 
-The repository defines a pattern to store DTDL interfaces in a directory structure based on the Digital Twin Model Identifier (DTMI). You can locate an interface in the repository by converting the DTMI to a relative path. For example, the DTMI `dtmi:com:example:Thermostat;1` translates to `/dtmi/com/example/thermostat-1.json`.
 
-## Examples
```

Comment:
Why is all this content being removed from the models repository SDK readme? The concepts and examples are meant to elevate devEx quickly bringing a user up to speed on client usage and pointing out some key service info.

Reply:
It was updated by https://github.com/Azure/azure-sdk-for-python/pull/24295/commits/dbdd3f6ed5ba647a209adc3c4dce12ec2841f796 because we did not opt out auto_update.

I have fixed the issue.